src/commands/scaling/scaleRule/ScaleRuleListStep.ts

```diff
@@ -18,6 +18,7 @@ export class ScaleRuleListStep extends AzureWizardPromptStep<ScaleRuleContext> {
     }
 
     private async getPicks(context: ScaleRuleContext): Promise<IAzureQuickPickItem<ScaleRule>[] | undefined> {
+        // Todo: Expand support for multiple revisions
```

Comment:
Don't think this is being used anywhere else. May want to add a comment on why we are still keeping this file.

Reply:
Maybe we should just remove it then and add it back in when we need it again?

Reply:
Sounds good to me!